src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -90,6 +90,12 @@ internal override async Task<AuthenticationResult> ExecuteAsync(CancellationToke
 
         private async Task AcquireAuthorizationAsync(CancellationToken cancellationToken)
```

Comment:
>cancellationToken [](start = 71, length = 17)

thinking: I don't believe we have any strategies for testing the cancellation token, just a plan on testing more... would it make sense to include one test here e.g. before calling AcquireAuthAsync?

Reply:
Yes, that's a good idea, can be done.